docs/source/concepts/functionality/command-line/index.rst

```diff
@@ -1,533 +1,478 @@
 .. dropdown:: Distribution Statement
 
- | # # # This source code is subject to the license referenced at
- | # # # https://github.com/NRLMMD-GEOIPS.
+ This source code is protected under the license referenced at
+ https://github.com/NRLMMD-GEOIPS.
```

Comment:
This source code is subject to the license referenced at
https://github.com/NRLMMD-GEOIPS.

Reply:
Fixed